src/main/java/net/consensys/orion/http/handler/set/SetPrivacyGroupRequest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package net.consensys.orion.http.handler.set;
+
+import net.consensys.orion.enclave.PrivacyGroupPayload;
+
+import java.io.Serializable;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Set a privacy group with the given id to the payload.
+ */
+public class SetPrivacyGroupRequest implements Serializable {
+  private final PrivacyGroupPayload payload;
+  private final String privacyGroupId;
```

Comment:
(not for here/now): Creating a strongly typed "PrivacyGroupIdentity" class (which can be serialised to a string) might make some of the business logic read more nicely, and give you somewhere for to put data validation - i.e. does it matter if the string supplied herein is not 32-bytes of Base64 encoded characters?